plugins/aws/s3/src/hopeit/aws/s3/object_storage.py

```diff
@@ -222,9 +226,9 @@ async def get(
         """
 
         async with self._session.client(S3, **self._conn_config) as object_storage:
+            key = f"{partition_key}/{key}" if partition_key else key
+            key = f"{self.prefix or ''}{key}"
```

Comment:
Should prefix always end with a / ?
we can ensure a / always by doing, also to make more straightforward to understand
that only something is prepended when is not None/empty:

```
if partition_key:
   key = f"{partition_key.strip('/')}/{key}"
if prefix:
   key = f"{self.prefix.strip('/')}/{key}"
```

Please check if partition_key and prefix do not come already stripped.

Reply:
I have created https://github.com/hopeit-git/hopeit.aws/issues/6 to address this.